### ISSUE TITLE
snakemake rule to clean results does not work on Windows 10

### ISSUE THREAD

**Issue by Stefanie08** [rl-institut/oemof-B3#323]
Upon entering the snakemake rule `snakemake -j1 clean` the following error message is printed:

```
INFO -
Der Befehl "rm" ist entweder falsch geschrieben oder
konnte nicht gefunden werden.
[Wed Mar 22 12:22:26 2023]
INFO - [Wed Mar 22 12:22:26 2023]
Error in rule clean:
    jobid: 0
    shell:

        rm -r ./results/*
        echo "Removed all results."

        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)
```

**Comment by MaGering** [comment]
Could be resolved adding a separate rule for windows users to the [snakefile](https://github.com/rl-institut/oemof-B3/blob/dev/Snakefile):

```
rule clean_on_win_sys:
    shell:
        """
        rmdir /s /q ./results/*
        echo "Removed all results."
        """
```

Please adapt so that all content (files and directories) are deleted in `./results` path. You'll need to adapt the windows shell command because `rmdir /s /q ./results/*` does not work yet.

**Comment by Stefanie08** [comment]
This issue is resolved with PR #329. Thus this issue will be closed.
